app/helpers/hosts_and_hostgroups_helper.rb

```diff
@@ -1,32 +1,13 @@
 module HostsAndHostgroupsHelper
   include AncestryHelper
+  include AccessibleResourceHelper
```

Comment:
Why are you including the helper again? The functions in ApplicationHelper are available on all pages.